index.html

```diff
@@ -29,38 +29,49 @@
     <link rel="icon" type="image/png" sizes="16x16" href="resources/favicon/favicon-16x16.png">
 
     <!-- Fonts -->
-    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+    <!-- Custom Win98 stylesheet -->
+    <link rel="stylesheet" href="https://unpkg.com/98.css" />
 </head>
 <body>
-    <nav>
-        <div class="left navbar-items">
-            <a id="logo" href="https://brickhack.io">
-                <img src="assets/logo.svg" alt="BrickHack 7 Logo" />
-            </a>
-            <a class="link" href="https://brickhack7.devpost.com/">DEVPOST</a>
-            <a class="link" href="./gallery.html">GALLERY</a>
-        </div>
-        <div class="right">
-            <a id="login" href="https://apply.brickhack.io/">LOGIN</a>
-            <a id="mlh-badge" href="https://mlh.io/seasons/na-2021/events?utm_source=na-hackathon&amp;utm_medium=TrustBadge&amp;utm_campaign=2020-season&amp;utm_content=gray">
-                <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2021/mlh-trust-badge-2021-gray.svg"  alt="Major League Hacking 2021 Hackathon Season" />
-            </a>
-        </div>
-    </nav>
-    <section id="end-main" class="section-pad">
-        <div id="thank-you-container">
-            <h1 class="end-h1">Thank you for attending BRICKHACK 7</h1>
-            <p>We hope to see you next year!</p>
-            <a href="./event.html">View the full event site</a>
+    <section id="main">
+
+        <img id="logo" src="./resources/pixel-logo-real.png">
+
+        <div class="window">
+            <div class="title-bar">
+                <div class="title-bar-text">BrickHack 8</div>
+                <div class="title-bar-controls">
+                    <button aria-label="Minimize"></button>
+                    <button aria-label="Maximize"></button>
+                    <button aria-label="Close"></button>
+                </div>
+            </div>
+            <div class="window-body">
+                <p>BrickHack 8 is returning in early 2022!</p>
+                <div class="field-row-stacked">
+                    <label for="text17">Enter your e-mail to stay updated.</label>
+                    <div class="field-row">
+                        <input id="text17" type="text" placeholder="hacker@email.com">
```

Comment:
Can this ID be changed to something more meaningful (e.g. "email", "emailField")?

Reply:
Yup, thought it was something on the stylesheet that I had to copy over